src/components/HeaderName/HeaderName.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import './HeaderName.scss'
+import { ReactComponent as User } from './icons/user.svg'
+
+type HeaderNameProps = {
+    firstName?: string,
+    lastName?: string,
+}
+
+export const HeaderName = ({firstName, lastName}: HeaderNameProps) => {
+    return (
+        <div className="header-name">
+            {!firstName ? 
```

Comment:
Тут лучше добавить дополнительную проверку
```suggestion
            {!firstName && lastName ? 
```